google-cloud-language/lib/google/cloud/language/annotation.rb

```diff
@@ -241,8 +241,8 @@ def self.from_grpc grpc
         #
         # @attr_reader [String] text The content of the output text.
         # @attr_reader [Integer] offset The API calculates the beginning offset
-        #   of the content in the original document according to the `encoding`
-        #   specified in the API request.
+        #   of the content in the original document according to the default
+        #   encoding of the system. Ruby defaults to UTF-8.
```

Comment:
We have that language in the guide documentation. I felt like it was distracting on the class documentation. My preference would be to remove this entire paragraph on the class documentation than pull it even more off-topic.